src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -85,6 +85,22 @@ const safeAppWeb3Provider = new Web3.providers.HttpProvider(getSafeAppsRpcServic
 const URL_NOT_PROVIDED_ERROR = 'App url No provided or it is invalid.'
 const APP_LOAD_ERROR = 'There was an error loading the Safe App. There might be a problem with the App provider.'
 
+// Some apps still need chain name, as they didn't update to chainId based SDK versions
+// With naming changing in the config service some names aren't the expected ones
+// Ex: Ethereum -> MAINNET, Gnosis Chain -> XDAI
+const getLegacyChainName = (chainName: string) => {
+  let network = chainName
+  switch (chainName) {
+    case 'Ethereum':
+      network = 'MAINNET'
+      break
+    case 'Gnosis Chain':
```

Comment:
Should we also map the other networks that we supported in the legacy sdk?

Reply:
The code snippet is returning the name used in the config service for all the other networks. Should be fine while it doesn't change. So we can decide if we can take care of this code when changing a network name or when adding it to the config service. I wan't to think that the name change is something that shouldn't happen that often

Reply:
Was mentioning this, as I think it would also be required for BSC and EWC :thinking: